scripts/uninstall-requirements.sh

```diff
@@ -1,15 +1,21 @@
 #!/bin/bash
-set -ev
-
-uninstall=requirements/uninstall-requirements.txt
+set -e
+uninstall=$1
+if [[ -z "${uninstall}" ]]
+then
+    uninstall=requirements/uninstall-requirements.txt
+fi
 tmp=requirements/uninstall-tmp.txt
 
-join <(grep -v '#' $uninstall | sort -n) <(pip freeze | grep -v '^-' | cut -d'=' -f1 | sort -n) > $tmp
-join <(grep -v '#' $uninstall | sort -n) <(pip freeze | grep -v '^-' | sort -n) >> $tmp
+join <(grep -v '#' ${uninstall} | sort -n) <(pip freeze | grep -v '^-' | cut -d'=' -f1 | sort -n) > ${tmp}
+join <(grep -v '#' ${uninstall} | sort -n) <(pip freeze | grep -v '^-' | sort -n) >> ${tmp}
 
-# if $tmp isn't just a single newline character
-if [ -s $tmp ]
+# if ${tmp} isn't just a single newline character
+if [ -s ${tmp} ]
 then
-    pip uninstall -r $tmp --yes
+    echo "Uninstalling from ${uninstall}"
+    pip uninstall -r ${tmp} --yes
+else
+    echo "Nothing to uninstall from ${uninstall}"
```

Comment:
I think this might be a holdover from an old pip that didn't handle empty files well or something. Do you know of a reason that we shouldn't remove this file and change `commcare-cloud` to do a simple `pip uninstall -r file --yes` ?

Output from my testing for reference. It works (and return code is 0) even when there's nothing to uninstall:

```
(commcare-hq) [16:37:58] jemord:commcare-hq git:(je/form-delete) $ pip uninstall -r requirements/uninstall-requirements.txt --yes
Uninstalling architect-0.5.6:
  Successfully uninstalled architect-0.5.6
Uninstalling django-nose-1.4.4:
  Successfully uninstalled django-nose-1.4.4
WARNING: Skipping feedparser as it is not installed.
WARNING: Skipping pycrypto as it is not installed.
WARNING: Skipping restkit as it is not installed.
WARNING: Skipping http-parser as it is not installed.
WARNING: Skipping zeep as it is not installed.
WARNING: Skipping cython as it is not installed.
WARNING: Skipping django-celery as it is not installed.
WARNING: Skipping numpy as it is not installed.
WARNING: Skipping ConcurrentLogHandler as it is not installed.
WARNING: Skipping xhtml2pdf as it is not installed.
Uninstalling celery-4.1.1:
  Successfully uninstalled celery-4.1.1
WARNING: Skipping subprocess32 as it is not installed.
WARNING: Skipping eventlet as it is not installed.
(commcare-hq) [16:38:19] jemord:commcare-hq git:(je/form-delete) $ echo $?
0
(commcare-hq) [16:38:22] jemord:commcare-hq git:(je/form-delete) $ pip uninstall -r requirements/uninstall-requirements.txt --yes
WARNING: Skipping architect as it is not installed.
WARNING: Skipping django-nose as it is not installed.
WARNING: Skipping feedparser as it is not installed.
WARNING: Skipping pycrypto as it is not installed.
WARNING: Skipping restkit as it is not installed.
WARNING: Skipping http-parser as it is not installed.
WARNING: Skipping zeep as it is not installed.
WARNING: Skipping cython as it is not installed.
WARNING: Skipping django-celery as it is not installed.
WARNING: Skipping numpy as it is not installed.
WARNING: Skipping ConcurrentLogHandler as it is not installed.
WARNING: Skipping xhtml2pdf as it is not installed.
WARNING: Skipping celery as it is not installed.
WARNING: Skipping subprocess32 as it is not installed.
WARNING: Skipping eventlet as it is not installed.
(commcare-hq) [16:38:24] jemord:commcare-hq git:(je/form-delete) $ echo $?                                                       
0
```

Reply:
Yeah as far as we can tell we did _use_ to do that before this (2014) PR: https://github.com/dimagi/commcare-hq/pull/2921, which added this script. I can't glean exactly what the problem was, but it does sound like pip may have used to return a non-zero return code when there was nothing to uninstall or something. Your test seems to support the idea that reverting back to the simpler command would be totally fine, so I'll try that out on the companion PR

Reply:
OK I took this suggestion on https://github.com/dimagi/commcare-cloud/pull/3243. I was going to be lazy and just leave it as it is, but then considering there's risk in every change and we no longer need this change, I just reverted `scripts/uninstall-requirements.sh` back to what it was before. Once everyone's had a chance to update commcare-cloud, we can delete it.